scripts/Dockerfile.integration_test

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:18.04
```

Comment:
please take a look at other Dockerfiles in our repos. particularly, for professional dockerfiles, you need to minimize the number of overlayFS layers. that's why most production dockerfiles have commands in the following format:
```
RUN cm1 && \
  cmd2 && \
  cmd3
```

Reply:
done